### PR TITLE
Extensibility wizard

### PR DESCRIPTION
**Pull request by qbalukom**

### CODE REVIEW

File: docs/extensibility/display-section.md [hunk 1]
```diff
@@ -712,79 +712,80 @@ The **Injections** section contains a list of objects that defines the display s
...
+  - name: Name
+    source: metadata.name
+    widget: Text
+  - name: Namespace
+    source: metadata.namespace
+    widget: Text
+  - name: status
+    widget: Badge
+    highlights:
+    positive: - 'OK'
```
Comment (akucharska) [inline docs/extensibility/display-section.md:783]: Indentation problem

File: docs/extensibility/wizard.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+The **general** section is required and contains basic information about the created resources and additional options.
+
+### Item parameters
+
+- **id** - _[required]_ - an identifier used to reference the wizard to trigger it's opening.
+- **resources** - _[required]_ - information about the resources created by the wizard. This is a key value map with values consisting of:
+  - **kind** - _[required]_ Kubernetes kind of the resource.
+  - **version** - _[required]_ API version used for all requests.
+  - **group** - API group used for all requests. Not provided for Kubernetes resources in the core (also called legacy) group.
```
Comment (akucharska) [inline docs/extensibility/wizard.md:32]: What about **multiple**

File: docs/extensibility/wizard.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+Each wizard consists of steps, this sections contains their definitions.
+
+### Item parameters
+
+Each step contains the following parameters.
+
+- **name** - _[required]_ - the name of the step displayed on the step navigation and in the step header.
+- **description** - extra details about the step, shown only when the step is active
+- **resource** - _[required]_ - identifier of the default resource for this step
+- **form** - _[required]_ - definition of the form - this is analogous to the contents of the [_form_ section](form-section.md) of the resource extension.
```
Comment (akucharska) [inline docs/extensibility/wizard.md:62]: We have **id** in form that is not mentioned in docs

File: docs/extensibility/wizard.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+Each wizard consists of steps, this sections contains their definitions.
+
+### Item parameters
+
+Each step contains the following parameters.
+
+- **name** - _[required]_ - the name of the step displayed on the step navigation and in the step header.
+- **description** - extra details about the step, shown only when the step is active
+- **resource** - _[required]_ - identifier of the default resource for this step
```
Comment (akucharska) [inline docs/extensibility/wizard.md:61]: There is also visibility parameter

File: public/wizard.yaml [hunk 5]
```diff
@@ -0,0 +1,84 @@
+apiVersion: v1
```
Comment (akucharska) [inline public/wizard.yaml:1]: Put it in examples

**Review by akucharska — COMMENTED** [review]
We also need to add schemas for a wizard

File: docs/extensibility/wizard.md (hunk 2 above)
Comment (akucharska) [reply docs/extensibility/wizard.md:32]: // flag for multiple resources of the same kind

File: docs/extensibility/wizard.md (hunk 3 above)
Comment (akucharska) [reply docs/extensibility/wizard.md:62]: // id would be used for refs

File: docs/extensibility/wizard.md (hunk 4 above)
Comment (mrCherry97) [reply docs/extensibility/wizard.md:61]: For whole sep visibility is not working, will be done as follow-up
https://github.com/kyma-project/busola/issues/2397

File: docs/extensibility/wizard.md (hunk 3 above)
Comment (mrCherry97) [reply docs/extensibility/wizard.md:62]: will be done as follow-up
https://github.com/kyma-project/busola/issues/2400

File: docs/extensibility/wizard.md (hunk 2 above)
Comment (mrCherry97) [reply docs/extensibility/wizard.md:32]: will be done as follow-up
https://github.com/kyma-project/busola/issues/2431

File: src/components/Extensibility/ExtensibilityWizard.js [hunk 6]
```diff
@@ -0,0 +1,242 @@
...
+              showValidity={true}
+              rootRule={prepareSchemaRules(step.form)}
+              onChange={actions => onChange(actions, step.resource)}
+            >
+              <FormStack
+                isRoot
+                schema={schemaMaps[step.resource]}
+                resource={resources[step.resource]}
+              />
+            </UIStoreProvider>
```
Comment (akucharska) [inline src/components/Extensibility/ExtensibilityWizard.js:194]:
```suggestion
            <section className="resource-form">
              <p>{step?.description}</p>
              <UIStoreProvider
                store={store[step.resource]}
                showValidity={true}
                rootRule={prepareSchemaRules(step.form)}
                onChange={actions => onChange(actions, step.resource)}
              >
                <FormStack
                  isRoot
                  schema={schemaMaps[step.resource]}
                  resource={resources[step.resource]}
                />
              </UIStoreProvider>
            </section>
```

Comment (akucharska) [reply src/components/Extensibility/ExtensibilityWizard.js:194]: Add wrapper with class `resource-form` or add styles that make rows align

**Review by akucharska — DISMISSED** [review]

**Review by IwonaLanger — APPROVED** [review]

**Review by akucharska — APPROVED** [review]

**Pull request merged**